tests/utils.py

```diff
@@ -233,7 +235,9 @@ def wait_genesis(genesis_time, genesis_delta):
         time.sleep(delta_from_genesis)
 
 
-def wait_for_minimal_elk_cluster_ready(namespace, es_ss_name=ES_SS_NAME, logstash_ss_name=LOGSTASH_SS_NAME):
+def wait_for_minimal_elk_cluster_ready(namespace, es_ss_name=ES_SS_NAME,
+                                                  logstash_ss_name=LOGSTASH_SS_NAME,
+                                                  kibana_ss_name=KIBANA_SS_NAME):
```

Comment:
ss stands for statefulset, Kibana is a deployment it should be named KIBANA_DEP_NAME

Reply:
same with kibana_ss_name